spark/src/test/scala/org/apache/spark/sql/delta/MergeIntoTestUtils.scala

```diff
@@ -96,18 +96,14 @@ trait MergeIntoSQLTestUtils extends DeltaSQLTestUtils with MergeIntoTestUtils {
       insert: String): Unit =
     sql(basicMergeStmt(target, source, condition, update, insert))
 
-  protected def mergeStmt(
-      target: String,
-      source: String,
-      condition: String,
-      clauses: MergeClause*): String =
-    s"MERGE INTO $target USING $source ON $condition\n" + clauses.map(_.sql).mkString("\n")
-
   override protected def executeMerge(
       tgt: String,
       src: String,
       cond: String,
-      clauses: MergeClause*): Unit = sql(mergeStmt(tgt, src, cond, clauses: _*))
+      clauses: MergeClause*): Unit = {
+    val clausesStr = clauses.map(_.sql).mkString("\n")
+    sql(s"MERGE INTO $tgt USING $src ON $cond\n$clausesStr")
```

Comment:
```suggestion
    sql(s"MERGE INTO $tgt USING $src ON $cond\n" + clausesStr)
```

I feel like this is slightly more readable.